cmd/go.mod

```diff
@@ -3,7 +3,7 @@ module github.com/awslabs/soci-snapshotter/cmd
 go 1.20
 
 require (
-	github.com/awslabs/soci-snapshotter v0.0.0-local
+	github.com/awslabs/soci-snapshotter v0.4.0
```

Comment:
This doesn't look right. Can we reduce the dependency changes down to just the ones needed for rootless?

It looks like there was a larger update here.